packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -74,6 +81,7 @@ const itemSeparator: ComponentSlotStyleFunction<MenuItemPropsAndState, MenuVaria
   const { iconOnly, pointing, pills, primary, underlined, vertical } = props
 
   return (
+    !vertical &&
     !pills &&
     !underlined &&
     !(pointing && vertical) &&
```

Comment:
do you still need the !vertical in line 87 since you added !vertical in 84?

Reply:
Good catch. No we don't.